lib/manageiq/messaging/client.rb

```diff
@@ -170,9 +170,10 @@ def subscribe_background_job(options)
       # Other options are underlying messaging system specific.
       #
       def publish_topic(options)
-        assert_options(options, [:event, :service])
+        messages = Array.wrap(options)
```

Comment:
Had to use `Array.wrap()` here from AS because `Array({:service => "foo"})` was turning into `[[:service, "foo"]]` not `[{:service => "foo"}]`

Reply:
Also `publish_topic(*messages)` wasn't handling `publish_topic(inventory_collection.map { |obj| {:whatever} })` aka an actual array passed in so this seemed to handle single hash or an array of hashes better than requiring the caller to splat the args out.  If you think this is too much overhead for the single hash case we could `kind_of?(Array) ? :`